src/htm/regions/ScalarEncoderRegion.cpp

```diff
@@ -205,6 +212,23 @@ ScalarEncoderRegion::~ScalarEncoderRegion() {}
                                   "",      // constraints
                                   "false", // defaultValue
                                   ParameterSpec::CreateAccess));
+  ns->parameters.add("sparsity",
+                    ParameterSpec(
+                                  "How sparce it is as a ratio of active bits to size."
```

Comment:
Typo: "Sparse" not "sparce"

Also, I usually phrase the description of sparsity differently, I prefer the word "fraction" instead of "ratio" because I think fraction is more understandable. But this might just be the notation I was taught in school.

Its the difference between:
`active-bits : total-size`
`active-bits / total-size`

Or maybe you could state the equation by saying:
"The sparsity is the number of active bits divided by the total number of bits"

Reply:
Thanks for the review.  Good comments as always.
I am making the changes.